app/services/nomis/elite2/prison_offender_manager.rb

```diff
@@ -3,7 +3,7 @@ module Elite2
     class PrisonOffenderManager
       include MemoryModel
 
-      attribute :staff_id, :string
+      attribute :staff_id, :integer
```

Comment:
I did not realise the value could be casted  👍

Reply:
Yeah, it was causing issues elsewhere when we were doing comparisons between integer ids and strings.